src/components/Article.js

```diff
@@ -0,0 +1,19 @@
+import React from "react"
+
+
+
+function Article({title, date ='January 1, 1970', preview}) {
+  return (
+    <>
+    <article>
+      <h3>{title}</h3>
+      <small>{date}</small>
+      <p>{preview}</p>
+    </article>
+    </>
+  )
+}
+
+
+
+export default Article
```

Comment:
When are fragments needed? Do you need one here?
```suggestion
import React from "react"

function Article({ title, date ='January 1, 1970', preview }) {
  return (
    <article>
      <h3>{title}</h3>
      <small>{date}</small>
      <p>{preview}</p>
    </article>
  )
}

export default Article
```